queries.sql

```diff
@@ -14,4 +14,44 @@ select * from animals where neutered = '1';
 select * from animals where not name='Gabumon';
 /* Find all animals with a weight between 10.4kg and 17.3kg 
 (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-select * from animals where weight_kg between 10.4 and 17.3;
\ No newline at end of file
+select * from animals where weight_kg between 10.4 and 17.3;
+
+/* Transaction 1 : update the animals table by setting the species column to unspecified.*/
+begin;
+Update animals set species = 'unspecified';
+rollback;
+
+/* Transaction 2 */
+begin;
+Update animals set species = 'digimon' where name like '%mon';
+Update animals set species = 'pokemon' where species is null;
+commit;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.), Please Verify that change was made and persists after commit.